README.md

```diff
@@ -51,10 +51,15 @@ end
 - path - the path, where your EmberCLI applications is located. The default
   value is the name of your app in the Rails root.
 
+- enable - a lambda that accepts each requests' path. The default value is a
+  lambda that returns `true`.
+
 ```ruby
 EmberCLI.configure do |c|
   c.app :adminpanel # path is "<your-rails-root>/adminpanel"
-  c.app :frontend, path: "/path/to/your/ember-cli-app/on/disk"
+  c.app :frontend,
+    path: "/path/to/your/ember-cli-app/on/disk",
+    enable: -> path { path.starts_with?("/app/") }
```

Comment:
I guess it's automatically implied. Anyone who sees a lambda passed in as an option should figure out it could be used for negative case by just inverting the logic.